splinterd/tests/admin/circuit_commit.rs

```diff
@@ -64,25 +64,19 @@ pub(in crate::admin) fn commit_2_party_circuit(circuit_id: &str, node_a: &Node,
     assert!(res.is_ok());
 
     // Wait for the proposal to be committed for the second node
-    let mut proposal_b;
```

Comment:
This commit might not be needed after #1284 goes in.

Reply:
Yep 👍

Reply:
I did have to add a commit to change the circuit's `management_type` so we don't have to skip through events or save an event ID if we attempt to commit multiple circuits. I found that as is, the `circuit_commit` function would receive old events from previously committed circuits (since the events are starting from the beginning) which would lead the test to run into issues. So, making that management type unique makes it so we can subscribe to the specific circuit's events.